src/hooks/useFarmStats.ts

```diff
@@ -27,87 +27,78 @@ function buildFarmStatsQuery(farmAddress: string) {
       totalUnlockedRewards
     }
   }
-`
+`;
 }
 
 type FarmStatsResponse = {
-  apy: BigIntish,
-  tvl: BigIntish,
-  staked: BigIntish,
-  bonusPeriodSec: number,
-  createdTimestamp: number,
-  totalUnlockedRewards: BigIntish
-}
+  apy: BigIntish;
+  tvl: BigIntish;
+  staked: BigIntish;
+  bonusPeriodSec: number;
+  createdTimestamp: number;
+  totalUnlockedRewards: BigIntish;
+};
 
 export function useFarmStats(farmAddress: string) {
-  const [apy, setAPY] = useState(ZERO_BIG_INT)
-  const [tvl, setTVL] = useState(ZERO_BIG_INT)
-  const [totalStaked, setTotalStaked] = useState(ZERO_BIG_INT)
-  const [rewardUnlockRate, setRewardUnlockRate] = useState(ZERO_BIG_INT)
-  const [daysLeft, setDaysLeft] = useState(0)
+  const [apy, setAPY] = useState(ZERO_BIG_INT);
+  const [tvl, setTVL] = useState(ZERO_BIG_INT);
+  const [totalStaked, setTotalStaked] = useState(ZERO_BIG_INT);
+  const [rewardUnlockRate, setRewardUnlockRate] = useState(ZERO_BIG_INT);
+  const [timeRemaining, setTimeRemaining] = useState({});
 
   const FARM_STATS_QUERY = buildFarmStatsQuery(farmAddress);
 
   useEffect(() => {
-    let cancelled = false
-    let retryTimer: NodeJS.Timeout
-    let farmDaysLeft = 0
+    let cancelled = false;
+    let retryTimer: NodeJS.Timeout;
```

Comment:
Is the type indeed a timeout from `NodeJS`? I hadn't seen that haha, thought we had a native browser type for `setTimeouts`